### PR TITLE
Do not use emphasis in conjunction with code font

### PR DESCRIPTION
**Pull request by jboxman**
Match:
```
`\*([A-Za-z]+)\*`
```

Replace:
```
`$1`
```

### CODE REVIEW

File: modules/ose_3_0_release_notes.adoc [hunk 1]
```diff
@@ -262,7 +262,7 @@ there.
 It is not possible to do correct JNDI lookup for datasources in the current JWS
 image if an invalid combination of datasource and realm properties is defined.
 If a datasource is configured in the *_context.xml_* file and a realm in the
-*_server.xml_* file, then the *_server.xml_* file's `*localDataSource*` property
+*_server.xml_* file, then the *_server.xml_* file's `localDataSource` property
```
Comment (kalexand-rh) [inline modules/ose_3_0_release_notes.adoc:265]: I'd revert the changes to file. It seems to be about 3.x and is using the correct styles for it.

**Review by kalexand-rh — COMMENTED** [review]
Aside from one 3.x file, this scans well. LGTM

File: modules/ose_3_0_release_notes.adoc (hunk 1 above)
Comment (jboxman) [reply modules/ose_3_0_release_notes.adoc:265]: @kalexand-rh, will do, thanks!
